core/src/main/scala/org/apache/spark/ui/exec/ExecutorsTab.scala

```diff
@@ -140,6 +140,8 @@ class ExecutorsListener(storageStatusListener: StorageStatusListener, conf: Spar
           return
         case _: ExceptionFailure =>
           taskSummary.tasksFailed += 1
+        case _: ExecutorLostFailure =>
```

Comment:
Hi,
   I have replaced most of the task failed cases with info.successful. Tested it as well. Have also written unit test to simulate my issue. Thank you.